BraintreeCore/src/main/java/com/braintreepayments/api/core/AnalyticsEvent.kt

```diff
@@ -8,5 +8,7 @@ internal data class AnalyticsEvent(
     val isVaultRequest: Boolean = false,
     val startTime: Long? = null,
     val endTime: Long? = null,
-    val endpoint: String? = null
+    val endpoint: String? = null,
+    val experiment: String? = null,
+    val buttonRank: Int? = null
```

Comment:
I think we moved forward with calling this `paymentMethodsDisplayed` as an array of strings: https://github.com/braintree/braintree_ios/blob/78c6c12030c94805602d2f1bdebf7c9b540152ef/Sources/BraintreeShopperInsights/BTShopperInsightsClient.swift#L110. Then we compact that down as a string with the key `payment_methods_displayed` to FPTI.

Reply:
I'll fix that up.